app/containers/SearchContainer/reducer.js

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * SearchContainer reducer
+ *
+ */
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+import get from 'lodash/get';
+
+export const initialState = { searchTerm: null, trackData: {}, trackError: null };
+
+export const { Types: searchContainerTypes, Creators: searchContainerCreators } = createActions({
+  requestTrackInfo: ['searchTerm'],
```

Comment:
this creator group fetches a list of tracks. So request trackInfo as a name is misleading. 

Maybe something like `requestGetTracks` would be appropriate here?